### DISCUSSION TITLE
Debug launch won't connect

### DISCUSSION THREAD

**Discussion by Cspenceb7** [microsoft/vscode-discussions#1070] [Extension Development QnA] [answered]
Good day. 
Brand new to this world. Trying to learn Autolisp. 
I am NOT an advanced computer user. 
I've successfully attached the Autocad Autolisp Extension. 
I have copied a few lines of code from a tutorial and hit the debug function... 
and gotten...
"C:\Program Files\Autodesk\AutoCAD 2024\acad.exe" doesn’t exist. Verify and correct the folder path to the product executable.

I have verified the location      and brick wall. 

Any thoughts would be most appreciated.

**Comment by alefragnani** [comment, marked as answer]
Hi @Cspenceb7 ,

I have no experience with the Autocad Autolisp Extension to help you (I would, if I had), but since you are having issues while using  this particular extension, I suppose the best place to look for info is on their repo (https://github.com/Autodesk-AutoCAD/AutoLispExt), since this Discussions repo is focused for extension development, for now.

I’m not sure you followed their setup instructions (https://github.com/Autodesk-AutoCAD/AutoLispExt#how-to-use-the-extension), worth to check, and a quick look on their issues, I found this comment (https://github.com/Autodesk-AutoCAD/AutoLispExt/issues/143#issuecomment-841131059). Maybe it is useful in your scenario.

Hope this helps

**Comment by Cspenceb7** [comment]
As Homer would say "doughhhhh"
A file address does not have"" marks. 
All better, but thanks.
